Pipfile

```diff
@@ -0,0 +1,28 @@
+[[source]]
+url = "https://pypi.python.org/simple"
+verify_ssl = true
+name = "pypi"
+
+[[source]]
+url = "https://pypi.pacificclimate.org/simple"
+verify_ssl = true
+name = "pcic"
+
+[packages]
+python-dateutil = "*"
+cdo = "==1.5.3"
+nchelpers = "==5.5.6"
+Pint = "*"
+PyYAML = "*"
+
+[dev-packages]
+pytest = "*"
+pytest-mock = "*"
+black = "==21.7b0"
```

Comment:
Why pin Black so very precisely, and not pin `pytest` at all?

Reply:
For some reason `pipenv` is unhappy when you don't specify a `black` version, so I just chose the latest one.

Reply:
oy vey